modules/tinymce/src/core/test/ts/browser/dom/DomUtilsTest.ts

```diff
@@ -721,7 +721,7 @@ describe('browser.tinymce.core.dom.DOMUtilsTest', () => {
     assert.isTrue(DOM.isEmpty(getTestElement()));
 
     DOM.setHTML('test', '<div><!-- comment --></div>');
-    assert.isFalse(DOM.isEmpty(getTestElement()));
+    assert.isTrue(DOM.isEmpty(getTestElement()));
```

Comment:
doesn't this change impact on some behaviors?

Reply:
yes, it is. that's why it is introduced in the major version.